src/main/java/slimeknights/tconstruct/smeltery/TinkerSmeltery.java

```diff
@@ -66,6 +68,7 @@
   public static final BuildingBlockObject searedRoad = BLOCKS.registerBuilding("seared_road", SMELTERY, TOOLTIP_BLOCK_ITEM);
   public static final BuildingBlockObject searedTile = BLOCKS.registerBuilding("seared_tile", SMELTERY, TOOLTIP_BLOCK_ITEM);
   public static final EnumObject<SearedTankBlock.TankType,SearedTankBlock> searedTank = BLOCKS.registerEnum("seared", SearedTankBlock.TankType.values(), (type) -> new SearedTankBlock(SMELTERY_GLASS), (b) -> new TankItem(b, SMELTERY_PROPS));
+  public static final BlockItemObject<FaucetBlock> searedFaucet = BLOCKS.register("faucet", () -> new FaucetBlock(SMELTERY.notSolid()), TOOLTIP_BLOCK_ITEM);
```

Comment:
`SMELTERY` is expected to be final, you should not be modifying it. Make a new property for the faucet. While properties are not stored in the block, its better to assume that each variable should be unique.

Reply:
Sure.

Reply:
Is something akin to
```java
private static final Block.Properties FAUCET = SMELTERY.notSolid();
```
valid in this case? Idea doesn't seem to care either way.

Reply:
Screw it. I'll just effectively dupe the property, like we do for glass. Would use glass,
but even though they're mechanically identical, semantically they are not, and they
may diverge over time.